statsmodels/iolib/tests/test_summary2.py

```diff
@@ -95,3 +95,59 @@ def test_OLSsummary(self):
 \begin{tabular}'''
         result = string_to_find in actual
         assert(result is True)
+
+
+class TestSummaryMixedLM(object):
+    # Test that MixedLM models are correctly summarized
+    def test_summary_fe(self):
+        # Test that the summary correctly includes all fixed effect variables.
+        pid = np.repeat([0, 1], 5)
+        x0 = np.repeat([1], 10)
+        x1 = [1, 5, 7, 3, 5, 1, 2, 6, 9, 8]
+        x2 = [6, 2, 1, 0, 1, 4, 3, 8, 2, 1]
+        y = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+        df = pd.DataFrame({"y": y, "pid": pid, "x0": x0, "x1": x1, "x2": x2})
+        endog = df["y"].values
+        exog = df[["x0", "x1", "x2"]].values
+        groups = df["pid"]
+        summ = MixedLM(endog, exog, groups=groups).fit().summary()
+
+        desired = ["const", "x1", "x2"]
+        actual = summ.tables[1].index.values[0:3]
+        assert_equal(actual, desired)
+
+    def test_summary_xname_fe(self):
+        # Test that the `xname_fe` argument is reflected in the summary table.
+        pid = np.repeat([0, 1], 5)
+        x0 = np.repeat([1], 10)
+        x1 = [1, 5, 7, 3, 5, 1, 2, 6, 9, 8]
+        x2 = [6, 2, 1, 0, 1, 4, 3, 8, 2, 1]
+        y = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
+        df = pd.DataFrame({"y": y, "pid": pid, "x0": x0, "x1": x1, "x2": x2})
+        endog = df["y"].values
+        exog = df[["x0", "x1", "x2"]].values
+        groups = df["pid"]
+        summ = MixedLM(endog, exog, groups=groups).fit().summary(xname_fe=["Constant", "Age", "Weight"])
+
+        desired = ["Constant", "Age", "Weight"]
+        actual = summ.tables[1].index.values[0:3] # Second table is summary of params, first three values are FE
```

Comment:
you could test all `summ.tables[1].index.values` each time
those should be deterministic

Reply:
Understood. I was not sure that the label `Group Var` for random effects would not change in the future, so I avoided including. Making the change now. Thanks for the feedback.

Reply:
The advantage of the tests is that there are no future unintentional changes that possibly break current functionality. If those future changes are intentional, then the unit tests will need to be adjusted when those changes are made.